src/test/java/se/citerus/dddsample/domain/model/cargo/DeliveryTest.java

```diff
@@ -24,51 +26,51 @@ public void testEvensOrderedByTimeOccured() throws Exception {
     Delivery dh = new Delivery(Arrays.asList(he1, he2, he3, he4));
 
     List<HandlingEvent> orderEvents = dh.history();
-    assertEquals(4, orderEvents.size());
-    assertSame(he2, orderEvents.get(0));
-    assertSame(he4, orderEvents.get(1));
-    assertSame(he1, orderEvents.get(2));
-    assertSame(he3, orderEvents.get(3));
+    assertThat(orderEvents).hasSize(4);
```

Comment:
Done.

As a first step, I used the automated script to make the changes. After that, I will use the AssertJ maven plugin to generate the Domain Driven Assertions. And as a final step, I should review all the assertions to exploit all the expressiveness of AssertJ.